app/routes/press-releases/press-release.js

```diff
@@ -2,17 +2,30 @@ import Route from '@ember/routing/route';
 import PressReleaseSnapshot from '../../utils/press-release-snapshot';
 
 export default class PressReleasesPressReleaseRoute extends Route {
+  from;
   async model(params) {
     const pressRelease = await this.store.findRecord('press-release', params.press_release_id, {
       include: [
+        'publication-channels',
         'publication-event'
       ].join(',')
     });
-    // include relations when more fields are added => e.g. sources.js route
-
     const snapshot = new PressReleaseSnapshot(pressRelease);
-    // await snapshot.commit();
-    // add snapshot.commit when relations are added
+    await snapshot.commit();
     return snapshot;
   }
+
+
+  beforeModel(transition) {
+    if(transition.from) {
+      this.from = transition.from.name;
+    } else {
+      this.from = 'press-releases.overview.concept'
+    }
+  }
+
+  setupController(controller, model) {
+    super.setupController(controller, model);
+    this.controllerFor('press-releases.press-release').set('from', this.from);
```

Comment:
`this.controllerFor('press-releases.press-release')` is de controller die je ook meekrijgt als argument in de `setupController`-functie.

`set()` is oude Ember syntax. In Ember Octane moet `from` als `@tracked` property gedefinieerd worden op de controller en kan die dan gewoon geassigned worden met `=` ipv `set()`. Ik zou dan ook opteren voor `fromRoute` ipv `from`.